src/fastoad/model_base/flight_point.py

```diff
@@ -232,17 +351,28 @@ def add_field(cls, name: str, annotation_type=float, default_value: Any = None,
         :param name: field name
         :param annotation_type: field type
         :param default_value: field default value
-        :param unit: expected unit for the added field ("-" should be provided for a dimensionless
-                     physical quantity)
+        :param unit: expected unit for the added field. "-" should be provided for a dimensionless
+                     physical quantity. Set to None, when unit concept does not apply.
+        :param cumulative: True if field value is sums up during mission
```

Comment:
Is summed up ?

Reply:
👍 
Fixed in a8495dfd0e9407b48464a4e65e2ec285a6e9c1bd